test/cloudtest/pkg/commands/main.go

```diff
@@ -670,6 +672,18 @@ func (ctx *executionContext) executeTask(task *testTask, clusterConfigs []string
 		task.test.Started = time.Now()
 		errCode := runner.Run(timeoutCtx, env, writer)
 
+		if errCode != nil {
+			if task.test.ExecutionConfig != nil {
+				onFailScriptErr := runOnFailScript(context.Background(), timeoutDuration, task.test.ExecutionConfig.OnFail, env, writer)
```

Comment:
Better to use runner.test.OnFailScript instead of task.test.ExecutionConfig.OnFail

Reply:
We have internally discussed. It is should be ok. But let me know if it still is a problem.

Reply:
fixed